app/cli/Main.hs

```diff
@@ -128,6 +135,11 @@ exec args
     | args `isPresent` command "address" && args `isPresent` command "list" = do
         wid <- args `parseArg` longOption "wallet-id"
         print (wid :: WalletId)
+        manager' <- newManager defaultManagerSettings
+        res <- runClientM (listAddresses (ApiT wid) (Just $ ApiT Unused)) (mkClientEnv manager' (BaseUrl Http "localhost" 8080 ""))
```

Comment:
We ought to create the connection manager before and pass it as argument I believe. In particular, the port address can be set via a CLI arg although, we don't have access to it at this stage :thinking: ... 
I see two options:

- Allow the port to be passed directly from the CLI for commands to (with default)
- Do some network discovery and try to find a corresponding wallet backend service

The former sounds more reasonable in the time-frame we have to finalize this.

Reply:
yes - I was about to opt for the former